test/test_runner_battle.c

```diff
@@ -812,6 +1148,7 @@ static s32 TryMessage(s32 i, s32 n, const u8 *string)
             continue;
 
         event = &DATA.queuedEvents[i].as.message;
+        // MgbaPrintf_("Looking for: %S Found: %S\n", event->pattern, string); // Useful for debugging.
```

Comment:
Did you mean to keep this? I'm guessing so based on the comment... I wonder if it would be helpful to have a define which turns on test debug logging :thinking:
```suggestion
        // MgbaPrintf_("Looking for: %S Found: %S\n", event->pattern, string); // Useful for debugging.
```

Reply:
Yeah, I'd like to keep it for now